herculens/MassModel/mass_model.py

```diff
@@ -75,12 +102,12 @@ def alpha(self, x, y, kwargs, k=None):
         :param k: only evaluate the k-th lens model
         :return: deflection angles in units of arcsec
         """
-        x = np.array(x, dtype=float)
-        y = np.array(y, dtype=float)
+        # x = np.array(x, dtype=float)
+        # y = np.array(y, dtype=float)
         if isinstance(k, int):
             return self.func_list[k].derivatives(x, y, **kwargs[k])
         bool_list = self._bool_list(k)
-        f_x, f_y = np.zeros_like(x), np.zeros_like(x)
+        f_x, f_y = 0., 0.
```

Comment:
this is a code comment: if x is an array the return object is not in the same format/dimension